homeassistant/components/logbook.py

```diff
@@ -286,6 +288,29 @@ def humanify(hass, events):
                     'context_user_id': event.context.user_id
                 }
 
+            elif event.event_type == EVENT_HOMEKIT_CHANGED:
+                data = event.data
+                entity_id = data.get(ATTR_ENTITY_ID)
+                dsp_name = data.get(ATTR_DISPLAY_NAME) or entity_id
+                service = data.get(ATTR_SERVICE) or ''
+                attribute = data.get(ATTR_VALUE)
+
+                attr_msg = " to {}".format(attribute) if attribute else ''
+                dsp_msg = " for {}".format(dsp_name) if dsp_name else ''
+
+                message = "send command {}{}{}".format(
+                    service, attr_msg, dsp_msg)
+
+                yield {
+                    'when': event.time_fired,
+                    'name': 'HomeKit',
+                    'message': message,
+                    'domain': 'homekit',
```

Comment:
Can you also import `'homekit'` from `homekit.const` with `from .. import DOMAIN as DOMAIN_HOMEKIT`?

Reply:
Done, should it be DOMAIN_HOMEKIT or HOMEKIT_DOMAIN? Asking as in logbook there is an import DOMAIN AS HA_DOMAIN from homeassistant.const.

Reply:
Doesn't really matter, as long as it is unique.